modules/social_features/social_lazy_loading/src/SocialLazyLoadingTextFormatOverride.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Drupal\social_lazy_loading;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+
+/**
+ * Class SocialLazyLoadingTextFormatOverride.
+ *
+ * @package Drupal\social_lazy_loading
+ */
+class SocialLazyLoadingTextFormatOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The module handler.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Constructs the configuration override.
+   *
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The configuration factory.
+   */
+  public function __construct(ModuleHandlerInterface $module_handler, ConfigFactoryInterface $config_factory) {
+    $this->moduleHandler = $module_handler;
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+
+    $formats = [
+      'basic_html' => TRUE,
+      'full_html' => TRUE,
+      'plain_text' => TRUE,
+      'simple_text' => TRUE,
+      'restricted_html' => TRUE,
+    ];
```

Comment:
Note that we are adding input formats this release (for e-mail and we might add comment) perhaps its better to just go through all the input formats programmatically

Reply:
Thanks, thats an excellent point, i will see what I can do here!